app.rb

```diff
@@ -0,0 +1,88 @@
+require './book'
+require './person'
+require './student'
+require './teacher'
+require './rental'
+
+class App
+  def list_all_books
+    Book.all.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_people
+    Person.all.each { |person| puts "Name: #{person.name}, Age: #{person.age}, ID: #{person.id}" }
+  end
+
+  def create_teacher
+    puts 'Name: '
+    name = gets.chomp
+    puts 'Age: '
+    age = gets.chomp
+    puts 'Specialization: '
+    specialization = gets.chomp
```

Comment:
- [ ] I couldn't also create a teacher because of the issue on the screenshot below:

![Screenshot from 2022-11-26 04-08-58](https://user-images.githubusercontent.com/63087491/204070673-98cb0a1f-7c15-45d5-af02-22525bdd7986.png)